DigitalLearningSolutions.Web/Controllers/MyAccountController.cs

```diff
@@ -415,5 +468,19 @@ DlsSubApplication dlsSubApplication
 
             return delegateAccount is { Active: true } ? delegateAccount : null;
         }
+
+        private async Task LogOutAndLogIntoCentrelessAccount(UserAccount userAccount)

```

Comment:
Rather than duplicating the logout and login logic, we should reuse the methods used by LoginController for logging users in and out.